Source/DiabloUI/credits.cpp

```diff
@@ -179,7 +180,7 @@ bool UiCreditsDialog()
 
 bool UiSupportDialog()
 {
-	if (gbIsHellfire) {
+	if (FindAsset("ui_art\\supportw.clx").ok()) {

```

Comment:
It'd be better to add an overload for `LoadOptionalClx(AssetRef &&ref)` to avoid looking up the file twice.